patches/android_webview-browser-permission-media_access_permission_request.cc.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/android_webview/browser/permission/media_access_permission_request.cc b/android_webview/browser/permission/media_access_permission_request.cc
+index ebf7f59d595171feb117eb76addbbb20f99b5633..d2d011a49d642d25ad536562580bdb243ff4e267 100644
+--- a/android_webview/browser/permission/media_access_permission_request.cc
++++ b/android_webview/browser/permission/media_access_permission_request.cc
+@@ -85,6 +85,7 @@ void MediaAccessPermissionRequest::NotifyRequestResult(bool allowed) {
+         audio_devices, request_.requested_audio_device_ids);
+     if (device)
+       devices.audio_device = *device;
++    if (/* DISABLES CODE */ (false))
```

Comment:
can we override `AwPermissionManager::SetOriginCanReadEnumerateDevicesAudioLabels` to be a no-op? Alternatively we could do something like `#define SetOriginCanReadEnumerateDevicesAudioLabels(URL, AUDIO) SetOriginCanReadEnumerateDevicesAudioLabels(URL, false)`?

Reply:
same for `SetOriginCanReadEnumerateDevicesVideoLabels` below

Reply:
cc: @AlexeyBarabash

Reply:
Thanks, I missed that opportunity.

Fixed at https://github.com/brave/brave-core/pull/26068/commits/bf0a1f61f4e3828cb4e5a8fd3ba03da9075f176f  .